R/estimate_and_apply_uncertainty.R

```diff
@@ -67,9 +72,9 @@ estimate_and_apply_uncertainty <- function(
     reporting_triangle,
     n_history_delay,
     n_retrospective_nowcasts,
+    structure = detect_structure(reporting_triangle),
```

Comment:
Any reason for this? I think technically to match the lower level function it should come after max delay

Reply:
have moved